src/13_file_io.py

```diff
@@ -8,11 +8,24 @@
 # Open up the "foo.txt" file (which already exists) for reading
 # Print all the contents of the file, then close the file
 
-# YOUR CODE HERE
+text = open('foo.txt')
+print(text.read())
+text.close()
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make 
 # sure that it contains what you expect it to contain
+print('\n\n')
+    
+new_file = open('bar.txt', 'w')
+new_file.write("I went to have an interview as a carpenter today")
+new_file.write("\nOh really? Well, good luck!")
+new_file.write("\nYeah, I think I really nailed it!")
```

Comment:
You can do it like this: `"I went to have an interview as a carpenter today \n Oh Really? Well, good luck \n Yeah, I think I really nailed it!"`
So then you don't need to have three lines.